ProductClass.js

```diff
@@ -0,0 +1,26 @@
+export class PRODUCT {
+  constructor(name, description, price, tags, images) {
+    this.name = name;
+    this.description = description;
+    this.price = price;
+    this.tags = tags;
+    this.images = images;
+    this.찜꽁 = "💖x 0"; //찜꽁
+  }
+  favorite() {
+    console.log(`${this.name}을(를) 찜 하였습니다.💖`);
+
+    let num = Number(this.찜꽁[3]);
+    num += 1;
+    this.찜꽁 = this.찜꽁.slice(0, 3) + num;
+  }
```

Comment:
여기도 앞서 말씀드린 리뷰와 동일한 내용입니다.
아래와  `this.favoriteCount = 0;`으로 수정하고 아래과 같이 개선이 필요해보입니다.
```suggestion
favorite() {
  this.favoriteCount += 1;
  console.log(`${this.name}을(를) 찜 하였습니다.💖 (현재: 💖x ${this.favoriteCount})`);
}
```